aggregator/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# This Dockerfile should be built with the root of the repo as context
+FROM 760097843905.dkr.ecr.eu-west-1.amazonaws.com/wellcome/sbt_wrapper:1.7.1 as sbt
+LABEL maintainer = "Wellcome Collection <digital@wellcomecollection.org>"
+
+WORKDIR /app
+COPY README.md ./
```

Comment:
Why is the README in the dockerfile?

Reply:
Mistake.  Something odd was going on and I put that in to prove I wasn't going mad.  I should have removed it.